scripts/jenkins-test

```diff
@@ -26,16 +25,24 @@ export SPARK_DRIVER_MEMORY=8g
 
 pushd "$ADAM_TMP_DIR"
 
+
 if [[ $HADOOP_VERSION == "1.0.4" ]]; then
```

Comment:
or possibly `[[ $HADOOP_VERSION =~ ^2\.6 ]]`, likewise below, if that seems easier